src/mlpack/methods/ann/layer/concat.hpp

```diff
@@ -194,6 +207,15 @@ class Concat
   void serialize(Archive& /* ar */, const unsigned int /* version */);
 
  private:
+  //! Parameter which indicates the input size of modules.
+  arma::Row<arma::sword> inputSize;
```

Comment:
See comment above.

Reply:
the inputSize may contain negative numbers, Since input size may be different for layers.
Eg (3,2,4) with (3,5,4) along axis 1, so the inputSize vector contains {3,-1,4}.

Reply:
I used arma::sword instead of int to comply with mlpack.

Reply:
We should use `size_t` since that is what is used to represent sizes everywhere else in mlpack.

Reply:
Sure, I have added it.